web/profiles/custom/os2loop/modules/os2loop_flag_content/src/Form/SettingsForm.php

```diff
@@ -52,6 +52,18 @@ public function buildForm(array $form, FormStateInterface $form_state) {
       '#default_value' => $config->get('to_email'),
     ];
 
+    $form['email_template'] = [
+      '#type' => 'textarea',
+      '#title' => $this->t('Email template for flag content body'),
+      '#default_value' => $config->get('template_body'),
+    ];
+
+    $form['subject_template'] = [
+      '#type' => 'textarea',
+      '#title' => $this->t('Subject template for flag content subject'),
+      '#default_value' => $config->get('template_subject'),
+    ];
```

Comment:
Both fields should be required.

Reply:
See https://github.com/itk-dev/os2loop/blob/develop/web/profiles/custom/os2loop/modules/os2loop_share_with_a_friend/src/Form/SettingsForm.php#L69-L98 for an example on how you can add info on tokens that are available in the templates.